### PR TITLE
When building oc from build-*-images, use found path

### PR DESCRIPTION
**Pull request by smarterclayton**
Don't assume PATH contains oc - oversight when merged.

@brenton addresses https://github.com/openshift/origin/pull/7823#issuecomment-217935958

[test]

### CODE REVIEW

**Comment by smarterclayton** [comment]
@mfojtik if you can review

**Comment by brenton** [comment]
This is odd, somehow https://ci.openshift.redhat.com/jenkins/job/test_pull_requests_origin_integration/789/consoleText doesn't have the 'build' function.

File: pkg/cmd/cli/cmd/dockerbuild/dockerbuild.go [hunk 1]
```diff
@@ -119,10 +120,16 @@ func (o *DockerbuildOptions) Run() error {
 		if glog.V(2) {
 			glog.Infof("Builder: "+format, args...)
 		} else {
-			fmt.Fprintf(e.ErrOut, "# %s\n", fmt.Sprintf(format, args...))
+			fmt.Fprintf(e.ErrOut, "--> %s\n", fmt.Sprintf(format, args...))
```
Comment (mfojtik) [inline pkg/cmd/cli/cmd/dockerbuild/dockerbuild.go]: +1 for this... I was going to suggest that in the other PR... '#' looks weird.

File: pkg/cmd/cli/cmd/dockerbuild/dockerbuild.go [hunk 2]
```diff
@@ -119,10 +120,16 @@ func (o *DockerbuildOptions) Run() error {
...
 		}
 	}
-	return stripLeadingError(e.Build(f, o.Arguments))
+	safe := interrupt.New(func(os.Signal) { os.Exit(1) }, func() {
+		glog.V(5).Infof("invoking cleanup")
+		if err := e.Cleanup(); err != nil {
+			glog.V(5).Infof("Cleanup error: %v", err)
```
Comment (mfojtik) [inline pkg/cmd/cli/cmd/dockerbuild/dockerbuild.go]: why we hiding this for the user by default?

**Comment by smarterclayton** [comment]
it's still not finding the binary even after it is built.  I think a path is missing.

**Comment by smarterclayton** [comment]
[merge]

**Comment by smarterclayton** [comment]
[merge]

**Comment by smarterclayton** [comment]
Debug flake https://github.com/openshift/origin/issues/8828

**Comment by smarterclayton** [comment]
[test]

**Pull request merged**